### ISSUE TITLE
No loader is configured for ".node" bcrypt.darwin-arm64.node

### ISSUE THREAD

**Issue by AndriP8** [napi-rs/node-rs#721]
I got an error like this
<img width="960" alt="Screenshot 2023-10-11 at 22 22 56" src="https://github.com/napi-rs/node-rs/assets/56879512/f3cd60db-e795-44a2-86ce-b321d0e81b7b">
 
i am using:

- nodejs v16.19.1
- nx for monorepo
- @node-rs/bcrypt v1.7.3
- chip M1

**Comment by hpohlmeyer** [comment]
I ran into the same issue with `@node-rs/argon2`. I could fix it by marking the package as external in my bundler.

__Vite__
```ts
// vite.config.ts

export default defineConfig({
  optimizeDeps: {
    exclude: ["@node-rs/argon2"],
  },
});
```

__esbuild__
```ts
import * as esbuild from 'esbuild'

await esbuild.build({
  external: ["@node-rs/argon2"]
  // other options …
})
```

**Comment by hpohlmeyer** [comment]
Yes, but if you deploy your production dependencies along-side your bundle this works fine. If you want a single bundled file, you are right, this approach definitely does not work.

**Comment by sep2** [comment]
Use `loader: { '.node': 'copy' }` this will copy the .node file to dist.
